x11-terms/tilda/tilda-1.5.4.ebuild

```diff
@@ -0,0 +1,32 @@
+# Copyright 1999-2022 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+inherit autotools flag-o-matic
+
+DESCRIPTION="A drop down terminal, similar to the consoles found in first person shooters"
+HOMEPAGE="https://github.com/lanoxx/tilda"
+SRC_URI="https://github.com/lanoxx/tilda/archive/${P}.tar.gz"
+
+LICENSE="GPL-2"
+SLOT="0"
+KEYWORDS="~amd64 ~arm64 ~ppc ~ppc64 ~x86 ~amd64-linux ~x86-linux"
+
+RDEPEND="x11-libs/vte:2.91
+	>=dev-libs/glib-2.8.4:2
+	dev-libs/confuse:=
+	gnome-base/libglade
+	x11-libs/gtk+:3
+	x11-libs/libX11"
+DEPEND="${RDEPEND}
+	virtual/pkgconfig
```

Comment:
These are both BDEPENDs usually (we need to execute them, not cross-compile them and use their headers or libraries).